client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -167,11 +167,12 @@ float getPenumbraRadius(sampler2D shadowsampler, vec2 smTexCoord, float realDist
 	}
 
 	vec2 clampedpos;
-	float texture_size = 2.0 /  f_textureresolution ;
+	// this is fixed to make sure same penumbra radius with all texture sizes
+	float texture_size = 2.0 /  2048 /*f_textureresolution*/ ;
 	float y, x;
-	float depth = 0.0;
+	float depth = 0.0, min_depth = 10000.0, max_depth = -10000.0;
 	float pointDepth;
-	float maxRadius = SOFTSHADOWRADIUS * 5.0 * multiplier;
+	float maxRadius = SOFTSHADOWRADIUS * 15.0 * multiplier;
 
 	float bound = clamp(PCFBOUND * (1 - baseLength), 0.5, PCFBOUND);
```

Comment:
Wouldn't it be better to clamp this to [0, PCFBOUND]? As is it always does a little bit of filtering (0.5).

Reply:
This clamping seriously damages distant shadows, see commit and comment below.